storm-server/src/main/java/org/apache/storm/LocalDRPC.java

```diff
@@ -38,9 +39,9 @@
     private final DRPC drpc;
     private final String serviceId;
 
-    public LocalDRPC() {
+    public LocalDRPC(StormMetricsRegistry metricsRegistry) {
```

Comment:
This feels problematic to me.

At a minimum we need to update ./docs/Local-mode.md to have the new way to create it documented, but I would really prefer to just have at least the option for a default constructor.  This is local mode the only reason we would need to have a single metrics registry would be if we intended to gather metrics from it afterwards as a part of a test, which we don't really do right now.

Reply:
Added a no-args constructor that creates a new registry.